rpctest/node.go

```diff
@@ -159,22 +146,16 @@ func (n *nodeConfig) rpcConnConfig() rpc.ConnConfig {
 
 // String returns the string representation of this nodeConfig.
 func (n *nodeConfig) String() string {
-	return n.prefix
+	return n.appDataDir
 }
 
 // cleanup removes the tmp data and log directories.
 func (n *nodeConfig) cleanup() error {
-	dirs := []string{
-		n.logDir,
-		n.dataDir,
-	}
-	var err error
-	for _, dir := range dirs {
-		if err = os.RemoveAll(dir); err != nil {
-			log.Printf("Cannot remove dir %s: %v", dir, err)
-		}
+	if err := os.RemoveAll(n.appDataDir); err != nil {
+		log.Printf("Cannot remove dir %s: %v", n.appDataDir, err)
+		return err
```

Comment:
don't see much point to logging the error and then returning it to the caller, who will probably end up logging it again.